### ISSUE TITLE
Save button in Web UI is unresponsive on latest dev

### ISSUE THREAD

**Issue by xoseperez** [xoseperez/espurna#1244]

**Comment by mcspr** [comment]
This form checking haunts me.

https://github.com/xoseperez/espurna/blob/ba4d4be8108ec843a0037db80055301dc3ae541c/code/html/custom.js#L173
evaluates to `true === 0` -> `false`

```js
var zeroLen = [adminPass1.length, adminPass2.length].every(function(val) { return val === 0; });
if (formValidity && zeroLen) {
```
Just to verify, this should be equivalent in real js (not python-like...)

**Comment by xoseperez** [comment]
Fixed in #1249
